app/javascript/components/concept-exercise-graph/ConceptGraph.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+
+import { Concept } from './Concept'
+import { ConceptConnections } from './ConceptConnections'
+
+import {
+  IConcept,
+  IConceptGraph,
+  ConceptLayer,
+  ConceptConnection,
+} from './concept-types'
+
+export const ConceptGraph = ({
+  concepts,
+  layout,
+  connections,
+}: IConceptGraph) => {
+  const [active, setActive] = useState<string | null>(null)
+
+  const conceptsBySlug = indexConceptsBySlug(concepts)
+  const adjacentBySlug = indexAdjacentBySlug(connections)
+
+  const adjacentConceptsToActive = new Set(
+    Array.from(document.getElementsByClassName(`adjacent-to-${active}`)).map(
+      (element) => (element as HTMLElement).dataset.conceptSlug
+    )
+  )
+
+  return (
+    <figure className="c-concept-graph">
+      <ConceptConnections connections={connections} activeConcept={active} />
+      <div className="track">
+        {layout.map((layer: ConceptLayer, i: number) => (
+          <div key={`layer-${i}`} className="layer">
+            {layer.map((conceptSlug) => {
+              const concept = conceptsBySlug.get(conceptSlug)
+              const isDimmed =
+                active !== null &&
+                active !== concept?.slug &&
+                !adjacentConceptsToActive.has(concept?.slug)
+
+              // TODO: fix this error typescript error since it _may_ return undefined
+              return !concept ? null : (
+                <Concept
+                  index={concept.index}
+                  slug={concept.slug}
+                  conceptExercise={concept.conceptExercise}
+                  uuidOfExercise={concept.uuidOfExercise}
```

Comment:
I don't think this is actually used anywhere. 

It's also worth noting that the concept has a uuid we can use if we need something distinct that's not a slug.

Reply:
Ya, for sure, just wasn't sure the plan to build links, so there is some extra data kicking around.

Reply:
Sweet :)

Reply:
Let's get rid of this and anything else that's not actively being used right now, to get to a minimum first pass.